playbooks/common/openshift-cluster/node_docker_ca.yml

```diff
@@ -0,0 +1,128 @@
+---
+- name: Configure CA certificate for secure registry
```

Comment:
This playbook is gone now that we're adding the OpenShift CA to the system trusted roots for each node.